docs-translations/zh-CN/api/process.md

```diff
@@ -1,27 +1,26 @@
 # 进程
 
-Electron 中的 `process` 对象 与 upstream node 中的有以下的不同点:
+> process 对象扩展.
 
-* `process.type` String - 进程类型, 可以是 `browser` (i.e. main process)
-  或 `renderer`.
-* `process.versions.electron` String - Electron的版本.
-* `process.versions.chrome` String - Chromium的版本.
-* `process.resourcesPath` String - JavaScript源代码路径.
-* `process.mas` Boolean - 在Mac App Store 创建, 它的值为 `true`, 在其它的地方值为 `undefined`.
+Process: [Main](../glossary.md#main-process), [Renderer](../glossary.md#renderer-process)
+
+Electron 的 `process` 对象是
+[Node.js `process` 对象](https://nodejs.org/api/process.html) 的扩展.
```

Comment:
I noticed that there are some inconsistent usage  English and Chinese punctuation in translation documents, specially period `.` and `。`. I think we'd better stick to use `。` for them. Could you please update this document accordingly？